tools/trimmomatic_validate.cwl

```diff
@@ -81,7 +81,7 @@ outputs:
            return rec
          }
  
-baseCommand: [java, -Xmx4G, -jar, /opt/Trimmomatic-0.38/trimmomatic-0.38.jar] 
+baseCommand: [-Xmx4G, -jar, /bin/trimmomatic.jar] 
```

Comment:
The entrypoint for this image is just `java` to allow for args like `-Xmx4G` passed to `java` not the jar.

Also the jar is symlinked to `/bin` for ease of calling